docs/core/compatibility/core-libraries/7.0/old-framework-support.md

```diff
@@ -0,0 +1,147 @@
+---
+title: ".NET 7 breaking change: Library support for older frameworks"
+description: Learn about the .NET 7 breaking change in core .NET libraries where core libraries packages are no longer supported on some older frameworks.
+ms.date: 09/29/2022
+---
+# Library support for older frameworks
+
+Installing the core libraries packages into projects whose target framework is older than the following versions is no longer supported:
+
+- .NET Framework 4.6.2
+- .NET 6
+- .NET Standard 2.0
+
+## Previous behavior
+
+The latest non-prerelease core libraries packages that were part of the ".NET 6" wave were supported when used from projects targeting .NET Framework 4.6.1 or later, .NET Core 3.1 and later, or .NET Standard 2.0 or later.
+
+## New behavior
+
+.NET 7 core libraries packages are supported for use in projects targeting .NET Framework 4.6.2 and later, .NET 6 and later, or .NET Standard 2.0 or later.
+
+## Version introduced
+
+.NET 7 Preview 1
+
+## Type of breaking change
+
+This change can affect [source compatibility](../../categories.md#source-compatibility) and [binary compatibility](../../categories.md#binary-compatibility).
+
+## Reason for change
+
+Building packages for all existing frameworks increases the complexity and size of a package. The following frameworks are now out-of-support and we no longer build packages for these frameworks:
+
+- .NET Framework 4.6.1
+- .NET Core 3.1
+- .NET 5
+
+## Recommended action
+
+- If your project is no longer being evolved and only maintained, simply don't upgrade the impacted packages.
+- If your project is actively developed, upgrade it to one of the following frameworks:
+
+  - .NET Framework 4.6.2
+  - .NET Core 6
+  - .NET Standard 2.0
+
+## Affected APIs
+
+The following packages no longer ship old frameworks:
+
+- Microsoft.Bcl.AsyncInterfaces
```

Comment:
Consider NuGet links or at the very least, treat these as namespaces with an inline code syntax.